openapi.yaml

```diff
@@ -11850,6 +11850,46 @@ paths:
         source: >
           linode-cli object-storage ssl-delete \
             us-east-1 example-bucket
+  /object-storage/transfer:
+    get:
+      operationId: getObjectStorageTransfer
+      x-linode-cli-skip: true
+      servers:
+      - url: https://api.linode.com/v4
+      summary: Object Storage Transfer View
+      description: |
+        The amount of outbound data transfer used by your account's Object Storage buckets.
+        Object Storage adds 1 terabyte of outbound data transfer to your data transfer pool.
+        See the [Object Storage Pricing and Limitations](/docs/guides/pricing-and-limitations/)
+        guide for details on Object Storage transfer quotas.
+
+      tags:
+      - Object Storage
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - object_storage:read_write
+      responses:
+        '200':
+          description: A paginated list of buckets you own.
```

Comment:
Does this need to have a description? The transfer description below should suffice. This endpoint is also not paginated.

Reply:
Our theme requires a description here. I've updated the description copy.